### ISSUE TITLE
Request for Baseline Training Code

### ISSUE THREAD

**Issue by BubbleDirk** [himashi92/Co-BioNet#14]
I hope this message finds you well. I have been working on a project that builds upon your methodology and have successfully utilized the open-sourced code you provided. However, I encountered some challenges when attempting to replicate the baseline results reported in your paper. Specifically, the baseline performance on LA or Pancreas dataset I obtained (i.e., only use 10% labeled data for training) is significantly lower than the results you have described.

To ensure that my implementation aligns accurately with your experiments, I was wondering if you could share the original baseline training code. Having access to this would greatly aid in reproducing the baseline results and furthering my research based on your work.

Thank you very much for your time and assistance.

**Comment by himashi92** [comment]
All the reproducible hyperparameters and the model weights are mentioned in the repo.
To reproduce LA 10%, use this: 
python train_cobionet_semi.py --dataset_name LA --labelnum 8 --lamda 0.7 --consistency 1.0 --mu 0.01 --t_m 0.4 --max_iteration 15000 --batch_size 4 --labeled_bs 2

**Comment by BubbleDirk** [comment]
I am very grateful for your assistance. I'm trying to understand how you achieved the results shown in Figure 2(d), which is around 55.06%, using VNet with only 10% labeled data and no unlabeled data. Could you please point me to the file containing the corresponding training code?

**Comment by himashi92** [comment]
The code for this specific experiment is not in this repo. However, you can modify the code given here to train VNet using only 10% of LA data: https://github.com/himashi92/Co-BioNet/blob/main/code/train_cobionet_sup.py.

**Comment by BubbleDirk** [comment]
I have indeed tried this, removing all modules and using only one vnet and the Dice supervised loss, with only random cropping for data augmentation. However, the results consistently remain around 20%, which has been puzzling me for a month. I am therefore writing to request your help. I would be extremely grateful if you could share the corresponding code.
